config/crd/bases/solr.apache.org_solrbackups.yaml

```diff
@@ -63,10 +63,14 @@ spec:
         description: SolrBackup is the Schema for the solrbackups API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
[-0] Ugh, these reformatting changes are unfortunate.  No way around them I take it?

Reply:
No workaround. It'll probably get fixed when we upgrade the controller-gen version, but we will see.